tests/tests_pytorch/trainer/connectors/test_checkpoint_connector.py

```diff
@@ -102,6 +102,28 @@ def test_hpc_max_ckpt_version(tmpdir):
     )
 
 
+def test_ckpt_for_fsspec():
+    """Test that the CheckpointConnector is able to write to fsspec file systems."""
+
+    model = BoringModel()
+    # hardcoding dir since `tmpdir` can be windows path
+    trainer = Trainer(default_root_dir="memory://test_ckpt_for_fsspec", max_steps=1)
+    trainer.fit(model)
+    trainer.save_checkpoint("memory://test_ckpt_for_fsspec/hpc_ckpt.ckpt")
+    trainer.save_checkpoint("memory://test_ckpt_for_fsspec/hpc_ckpt_0.ckpt")
```

Comment:
Windows jobs are not happy about these slashes

Reply:
I do that to fix tests on windows. fsspec wants / as sep on all platforms.

Reply:
I don't have windows testing environment,so I have to push and run tests on GitHub actions. This change is to make fsspec works on windows (we don't want \ in s3 or hdfs paths), but it seems there is another error. I'll fix that later